hana/lib/cql-functions.js

```diff
@@ -9,6 +9,7 @@ const StandardFunctions = {
       ? `substring( ${x}, case when ${y} < 0 then length(${x}) + ${y} + 1 else ${y} + 1 end, ${z} )`
       : `substring( ${x}, case when ${y} < 0 then length(${x}) + ${y} + 1 else ${y} + 1 end )`,
   count: x => `count(${x || '*'})`,
+  countdistinct: x => `count(distinct ${x || '*'})`,
```

Comment:
is this intentionally part of this change? → PR title doesnt say anything bout a new function

Reply:
Yes it is.

There's a change in the runtime implementation for `countdistinct` which requires this in the new HANA Service.
Bob initially created #311 but I took over this change to have only one PR.
Sorry, for not mentioning this in the initial descrpition.

Reply:
Alright, since this is only in the hana channel, it won't be necessary to add it to the changelog, yet